src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,44 @@
 
 class Animal {
   // write your code here
+  static alive = [];
```

Comment:
The `alive` array is used to track living animals. Ensure that any logic that might affect an animal's health to zero, other than the `bite` method, also removes the animal from this array.